db/app.js

```diff
@@ -1,24 +1,33 @@
 const express = require('express');
-const {getTopics, getEndPoints} = require('../controllers/controllers.js');
+const {getTopics, getEndPoints, getArticleById} = require('../controllers/controllers.js');
 
 
 const app = express();
 
 app.get('/api/topics', getTopics);
 
 app.get('/api', getEndPoints);
+//4
+app.get('/api/articles/:article_id', getArticleById);
 
 
 
 app.all('/*',(request, response) =>{
-    console.log(response)
     response.status(404).send({ message: 'path is not found'})
   })
 
 //Error handling
 app.use((err, req, res, next) => {
+  if (res.statusCode === 404) {
+    res.status(404).send({ message: "item is not found" })
+  }
```

Comment:
Since we have sent a promise reject with a status code and a message this should be pciked up by your if(err.status)